src/Compilers/Core/Portable/Emit/EmitOptions.cs

```diff
@@ -26,11 +26,9 @@ public sealed class EmitOptions : IEquatable<EmitOptions>
 
         /// <summary>
         /// Unless set (private) members that don't affect the language semantics of the resulting assembly will be excluded
-        /// when emitting with <see cref="EmitMetadataOnly"/> on. 
+        /// when emitting metadata-only assemblies (either as primary output, with <see cref="EmitMetadataOnly"/> on, or as
+        /// secondary output.
```

Comment:
If you specify a secondary output, we currently require that you set this option to false. 
At this point, we don't have a scenario with the secondary output being metadata-only-including-private-members.

I can update the comment to clarify.

---
In reply to: [109529052](https://github.com/dotnet/roslyn/pull/18339#discussion_r109529052) [](ancestors = 109529052)